Source/WebCore/PlatformGTK.cmake

```diff
@@ -107,7 +108,7 @@ list(APPEND WebCore_SYSTEM_INCLUDE_DIRECTORIES
     ${UPOWERGLIB_INCLUDE_DIRS}
 )
 
-if (USE_OPENGL)
+if (USE_OPENGL AND NOT USE_LIBEPOXY)
```

Comment:
Since libexposy is required, we can just remove the if entirely. And conditional includes in the files that are specific to GTK and/or WPE.

Reply:
I would like to do this separately, go through `USE_LIBEPOXY` and remove disabled code. But it requires understanding what's specific to GTK and what's also affecting other ports.